.test_patterns.yml

```diff
@@ -227,7 +227,7 @@ tests:
   #     270 |
   #     271 |       const publicLogs = (await archiver.getPublicLogs({ fromBlock: blockNumber, toBlock: blockNumber + 1 })).logs;
   #     272 |       const expectedTotalNumPublicLogs = 4 * (blockNumber + 1) * 2;
-  - regex: "archiver/src/archiver.test.ts"
```

Comment:
I've updated this to point at https://github.com/AztecProtocol/aztec-packages/blob/bump-noir/yarn-project/archiver/src/archiver/archiver.test.ts as it looks like this file moved (or the path was set up incorrectly).

fyi @LeilaWang as you're assigned to this flake.